active/InputValidation/InputValidation.md

```diff
@@ -0,0 +1,572 @@
+            Windows.UI.Xaml.Controls.InputValidationCommand ValidationCommand;
+         };
+   
+## InputValidationCommand
+The command allows for easy use of input validation without the need for a tight dependency on x:Bind.  This enables developers who aren't striclty using MVVM to use input validation (i.e ReactiveUI).  The InputValidationCommand is what allows for the correct-by-design approach where controls initially validate on focus lost, but then will query for validation on property changed once in an invalid state.  This wouldn't be for free, but the amount of code here is pretty trivial. We could provide some attribute or pattern that tells the markup compiler how to generate that code automatically.  
+
+The command can shared between different IInputValidationControls, as would most likely be the case in the scenario of a forms control.  A custom command can be made that listens for changes to all errors and knows when a form has been completed and ready for submition. The control will query the command as to whether they should perform validation.
+
+Developers can override any of this behavior by creating their own InputValidationCommand.
+ 
+ 
+    /// <summary>
+    /// Command that controls how an IInputValidationControl does validation.
+    /// </summary>
+    [webhosthidden]
+    unsealed runtimeclass InputValidationCommand
+    {
+        [method_name("CreateInstance")] InputValidationCommand();
+        Windows.UI.Xaml.Controls.InputValidationKind InputValidationKind;
+        /// <summary>
+        /// Called by a control to determine if it should call the Validate method.
+        /// </summary>
+        Boolean CanValidate(Windows.UI.Xaml.Controls.IInputValidationControl validationControl);
+        /// <summary>
+        /// If CanValidate returns true, Validate is invoked to perform validation on the
+        /// validationControl provided.
+        /// </summary>
+        void Validate(Windows.UI.Xaml.Controls.IInputValidationControl validationControl);
+        /// <summary>
+        /// Method that derived classes implement to provide their own implementation of the CanValidate method.
+        /// </summary>
+        overridable Boolean CanValidateCore(Windows.UI.Xaml.Controls.IInputValidationControl validationControl);
+        /// <summary>
+        /// Method that derived classes implement to provide their own implementation of the Validate method.
+        /// </summary>
+        overridable void ValidateCore(Windows.UI.Xaml.Controls.IInputValidationControl validationControl);
+    };
+
+## ValidationContext
+The `ValidationContext` class provides context as to what is being required by validation. This class is auto-generated by the XAML markup compiler in certain scenarios (see [IInputValidationControl]()). Controls can use the provided properties on the `ValidationContext` to customize their appearance.  For example, the IsInputRequired field is can allow controls to put the little * next to a header. 
```

Comment:
For example, the IsInputRequired field ~is~ can allow controls to put the little * next to a header.